libbeat/template/template.go

```diff
@@ -210,20 +205,21 @@ func (t *Template) Generate(properties common.MapStr, dynamicTemplates []common.
 		indexSettings.Put("number_of_routing_shards", defaultNumberOfRoutingShards)
 	}
 
-	if t.esVersion.IsMajor(7) {
+	var mappingName string
+	major := t.esVersion.Major
+	switch {
+	case major < 6:
```

Comment:
I wonder if we should even allow to run Beats 7.0 against ES 5.x?

@monicasarbu @tsg